ramls/marccat.raml

```diff
@@ -150,3 +152,26 @@ resourceTypes:
           body:
             text/plain:
               example: "Error in validation"  
+
+  /auth-header-types:
+    displayName: Get the header type for 008 authority tag
+    type:
+      get-only:
+        schema: authorityHeader008Tag.json
```

Comment:
This should be just "authorityHeader008Tag"

i.e. as declared in the "types" section at line 19

Reply:
That's right. Thanks a lot.